js_modules/dagster-ui/packages/ui-core/src/app/AppProvider.tsx

```diff
@@ -44,6 +44,8 @@ import './blueprint.css';
 // when you copy-paste so they don't get pasted into editors, etc.
 patchCopyToRemoveZeroWidthUnderscores();
 
+const httpStatusCodesToRetry = [502, 503, 504, 429, 409];
```

Comment:
one concern is that for `429` we send a `retry-after` header which we certainly don't  respect here in our code, not sure if `RetryLink` under the hood will

Reply:
It won't under the hood but we can customize RetryLink to do this.

Reply:
We could do that by changing `delay` to a function and I *think* the `error` argument should have the header accessible https://www.apollographql.com/docs/react/api/link/apollo-link-retry/#custom-strategies